ersilia/cli/commands/catalog.py

```diff
@@ -88,10 +109,13 @@ def catalog(
                 )
             )
             return
-        if file_name is None:
-            catalog = catalog_table.as_json()
+        if as_table:
```

Comment:
I think this logic is between whether we want to show the catalog on the screen vs save it to a file. So, we should indeed have the following flow:

```Python
if file_name is None:
     catalog = catalog_table.as_table() if as_table else catalog_table.as_json()
else:
     catalog_table.write(file_name)
     catalog = None
```

Reply:
@DhanshreeA Thanks, I've implemented this is the latest commit